proto/types.proto

```diff
@@ -153,6 +157,10 @@ message FirecrackerDriveMount {
   // writes to the image from within the guest. Defaults to false, in which case
   // the block device in the VM will be read-only.
   bool IsWritable = 6;
+
+  // (Optional) CacheType specifies the caching strategy for the block device.
+  // The supported caching strategies are: "Unsafe"(default) and "WriteBack".
```

Comment:
The canonical value of `DriveCacheTypeWriteback` is `"Writeback"`, not `"WriteBack"`.  The API may not be case-sensitive, but I haven't tested that theory yet.

Reply:
Good catch. I will change the comment to  have the correct string.

Reply:
Yeah, the API does not appear to be case-sensitive.
```
--- FAIL: TestDriveMountFails_Isolated (0.06s)
    service_integ_test.go:1242: 
                Error Trace:    service_integ_test.go:1242
                Error:          Received unexpected error:
                                rpc error: code = Unknown desc = failed to create VM: failed to start the VM: [PUT /drives/{drive_id}][400] putGuestDriveByIdBadRequest  &{FaultMessage:An error occurred when deserializing the json body of a request: unknown variant `WriteBack`, expected `Unsafe` or `Writeback` at line 1 column 25.}
                Test:           TestDriveMountFails_Isolated
FAIL
```